src/test/java/org/jenkinsci/test/acceptance/recorder/TestRecorderRuleTest.java

```diff
@@ -105,4 +120,23 @@ private TestRecorderRule rule(Description desc) {
     private File outputFile(Description desc) {
         return new File("target/diagnostics/" +desc + "/ui-recording.mov");
     }
+
+    private String updateRecorderToDefaultOption() {
+        final String envVar = System.getenv("RECORDER");
+        final String recorder = System.getProperty("RECORDER");
```

Comment:
This is ugly, we should not be allowing _both_ environment variables and system properties IMO.